dist/htmlminifier.js

```diff
@@ -1698,7 +1698,7 @@
     return minify(value, options);
   };
 
-}(typeof exports === 'undefined' ? this : exports));
+})(typeof exports === 'undefined' ? this : exports);
```

Comment:
I don't mind which one, I just thought we were using outside already. So, just keep it as you have it here.